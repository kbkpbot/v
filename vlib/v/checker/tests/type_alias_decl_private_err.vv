import cli

fn my_sort(a voidptr, b voidptr) int {
	return 1
}

fn hello(func FnSortCB) {
}

fn main() {
	_ := cli.Defaults(true)
	hello(my_sort)
}
