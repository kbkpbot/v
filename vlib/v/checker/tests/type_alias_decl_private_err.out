type_alias_decl_private_err.vv:7:15: error: module `builtin` function type `FnSortCB` is private
    5 | }
    6 |
    7 | fn hello(func FnSortCB) {
      |               ~~~~~~~~
    8 | }
    9 |
type_alias_decl_private_err.vv:11:11: error: module `cli` type `cli.Defaults` is private
    9 |
   10 | fn main() {
   11 |     _ := cli.Defaults(true)
      |              ~~~~~~~~~~~~~~
   12 |     hello(my_sort)
   13 | }

